org.speed.dreams.yaml

```diff
@@ -0,0 +1,166 @@
+  
+  - name: libxmu
+    sources:
+      - type: archive
+        url: https://www.x.org/archive//individual/lib/libXmu-1.1.2.tar.bz2
+        sha256: 756edc7c383254eef8b4e1b733c3bf1dc061b523c9f9833ac7058378b8349d0b
+  
+  - name: plib
+    rm-configure: true
+    config-opts:
+      - CXXFLAGS=-fPIC
+    sources:
+      - type: archive
+        url: http://plib.sourceforge.net/dist/plib-1.8.5.tar.gz
+        sha1: c2cf7e3e1e58f7b63dae4bb21e4fa82c3e4d4cfc
+        
+      - type: script
+        dest-filename: autogen.sh
+        commands:
+          - autoreconf -vfi
+
+  - name: enet
+    sources:
+      - type: archive
+        url: http://enet.bespin.org/download/enet-1.3.13.tar.gz
+        sha512: 82501c0248330a3a6bcac907907760e59b18cb662ebb42fe2c91e1557f1bc8ab452bf3fe7477dc01fa7a6e3b6124e1eb8eac46382d5bbebb2394879f9568b5df
+  
+  - name: freealut
+    rm-configure: true
+    cleanup:
+      - /bin
+    sources:
+      - type: archive
+        url: http://pkgs.fedoraproject.org/repo/pkgs/freealut/freealut-1.1.0.tar.gz/e089b28a0267faabdb6c079ee173664a/freealut-1.1.0.tar.gz
+        sha256: 60d1ea8779471bb851b89b49ce44eecb78e46265be1a6e9320a28b100c8df44f
+      - type: script
+        dest-filename: autogen.sh
+        commands:
+          - autoreconf -vfi
+          
+  - shared-modules/udev/udev-175.json
+  
+  - name: speed-dreams-2
+    buildsystem: cmake
+    config-opts:
+      - -DCMAKE_BUILD_TYPE=Release
+      - -DCMAKE_INSTALL_PREFIX=/app
+      - -DSD_BINDIR=/app/bin
+      - -DSD_DATADIR=/app/share/speed-dreams-2
+      - -DSD_LIBDIR=/app/lib/speed-dreams-2
+      - -DOPTION_OFFICIAL_ONLY=ON
+    
+    sources:
+      - type: file
+        url: https://downloads.sourceforge.net/sourceforge/speed-dreams/speed-dreams-src-base-2.2.2-rc2-r6527.tar.xz
+        md5: 956bda0acbcb89eaa2ca742dbedc8941
+        
+      - type: file
+        url: https://downloads.sourceforge.net/sourceforge/speed-dreams/speed-dreams-src-unmaintained-2.2.2-rc2-r6527.tar.xz
+        md5: 5d55831ca9371f8a55cc5616f9612164
+          
+      - type: file
+        url: https://downloads.sourceforge.net/sourceforge/speed-dreams/speed-dreams-src-wip-cars-and-tracks-2.2.2-rc2-r6527.tar.xz
+        md5: bc183b3e80c742bc249f630cb029920c
+          
+      - type: file
+        url: https://downloads.sourceforge.net/sourceforge/speed-dreams/speed-dreams-src-hq-cars-and-tracks-2.2.2-rc2-r6527.tar.xz
+        md5: ed11647ad2d479dc4012ffd79375b995
+          
+      - type: file
+        url: https://downloads.sourceforge.net/sourceforge/speed-dreams/speed-dreams-src-more-hq-cars-and-tracks-2.2.2-rc2-r6527.tar.xz
+        md5: d615eda2c443a2e5ce74cf45d5374062
+        
+      - type: file
+        path: org.speed.dreams.desktop
+        
+      - type: file
+        path: org.speed.dreams.appdata.xml
+        
+      - type: shell
+        commands:
+          - tar xf speed-dreams-src-base-2.2.2-rc2-r6527.tar.xz
```

Comment:
Flatpak already takes care of this, this will just double the work.

Reply:
It's necessary, otherwise the files don't extract on the corrects folders